src/editors/containers/VideoEditor/hooks.js

```diff
@@ -0,0 +1,80 @@
+import { useState } from 'react';
```

Comment:
I chose to place the logic at this level because of validateEntry. This is what checks for errors when we hit the Save button. This gets passed to the EditorContainer. The errors themselves can get passed down through the components from this level.